### PR TITLE
Add `SymIntElements` to enable unpacking `SymIntArrayRef`

### PR DESCRIPTION
**Pull request by miladm**
Add `SymIntElements` to enable unpacking `SymIntArrayRef`

### CODE REVIEW

File: torch_xla/csrc/torch_util.cpp [hunk 1]
```diff
@@ -5,6 +5,29 @@
...
+    size_nodes.push_back(size_node);
+    upper_bounds.push_back(
+        std::dynamic_pointer_cast<torch::lazy::DimensionNode>(size_node)
+            ->getStaticValue());
+    dynamic_dims.push_back(
+        std::dynamic_pointer_cast<torch::lazy::DimensionNode>(size_node)
+            ->isDynamic());
+  } else {
+    auto size_node = torch::lazy::MakeNode<Constant>(std::move(
+        XlaHelpers::ScalarLiteral(size.expect_int(), xla::PrimitiveType::F64)));
```
Comment (JackCaoG) [inline torch_xla/csrc/torch_util.cpp:25]: why f64? if it is a size then we should store it as s32.

File: torch_xla/csrc/torch_util.h [hunk 2]
```diff
@@ -5,9 +5,33 @@
...
+  SymIntElements(c10::SymIntArrayRef& size) {
+    std::vector<c10::SymInt> _sizes = torch::lazy::ToVector<c10::SymInt>(size);
+    for (auto& _size : _sizes) {
+      SetSymIntNodeElements(_size);
+    }
+  }
+
+  std::vector<torch::lazy::NodePtr> size_nodes;
+  std::vector<int64_t> upper_bounds;
+  std::vector<bool> dynamic_dims;
```
Comment (JackCaoG) [inline torch_xla/csrc/torch_util.h:29]: Can we renamed name with `_` at the end and move all of them to private then provide a getter(I don't think we need a setter).

**Review by JackCaoG — CHANGES_REQUESTED** [review]
nits

File: torch_xla/csrc/torch_util.h (hunk 2 above)
Comment (miladm) [reply torch_xla/csrc/torch_util.h]: We likely need setters. I will push setters at a later stage when the use for them is evident.

**Review by JackCaoG — APPROVED** [review]
Thanks!

**Pull request merged**